Tasks/DeployVisualStudioTestAgent/README.md

```diff
@@ -1,2 +1,27 @@
-# Deploy Visual Studio Agent Task
-<br/>
\ No newline at end of file
+# Deploy Visual Studio Test Agent Task
+
+###Overview
+The Deploy Test agent task is used to Deploy the Test Agent to a set of machines. This Test Agent can then be used to enable data collection or run Distributed Tests using the ‘Visual Studio Test using Test Agent’ task. 
```

Comment:
Prereqs section of run tests also applies here (dotnet4.5 and ps remoting). Please link or copy paste